bundles/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/AccountsServlet.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.groheondus.internal;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.http.HttpStatus;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ServiceScope;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Florian Schmidt - Initial contribution
+ */
+@NonNullByDefault({})
+@Component(service = AccountsServlet.class, scope = ServiceScope.SINGLETON)
+public class AccountsServlet extends HttpServlet {
+    private static final long serialVersionUID = -9183159739446995608L;
+
+    private static final String SERVLET_URL = "/groheondus";
+
+    private final Logger logger = LoggerFactory.getLogger(AccountsServlet.class);
+    private final List<Thing> accounts = new ArrayList<>();
+    @Reference
+    private HttpService httpService;
```

Comment:
```suggestion
    private @NonNullByDefault({}) HttpService httpService;
```

or use constructor injection (see below).

Reply:
Done!